code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -15,23 +15,8 @@
 	layer = LARGE_MOB_LAYER //Looks weird with them slipping under mineral walls and cameras and shit otherwise
 
 /mob/living/simple_animal/hostile/megafauna/death(gibbed)
-	if(health > 0)
-		return
-	else
-		feedback_set_details("megafauna_kills","[initial(name)]")
-		..()
-
-/mob/living/simple_animal/hostile/megafauna/gib()
```

Comment:
I just tested in a local server, I was easily able to get colossus to walk straight into a supermatter shard by placing it between us. It still drops it's loot.

I also tested with TK and you can just chuck the damn thing at anyone you want.